src/services/LbpProjectTokenPriceService.ts

```diff
@@ -0,0 +1,144 @@
+import { autoinject } from "aurelia-framework";
+import * as moment from "moment-timezone";
+import Moment = moment.Moment;
+
+@autoinject
+export class LbpProjectTokenPriceService {
+
+  maxSupply: number;
+  projectTokenInitialLbpAmount: number;
+  fundingTokenInitialLbpAmount: number;
+  startWeightProjectToken: number;
```

Comment:
why specify "ProjectToken"?  Is there any other kind of weight?  Feel free to document...

Reply:
I took it from Fabi's code, and thought the same too at the begining. But I guess it is more prominent to say that it's the projectToken weight in the pair - (80% PT) <-> (20% FT)